src/components/NpsSurvey.js

```diff
@@ -109,7 +110,7 @@ export const NpsSurvey = Utils.connectAtom(authStore, 'authState')(class NpsSurv
         onClick: () => this.setState({ expanded: true }),
         disabled: expanded,
         style: {
-          height: expanded ? 325 : 100,
+          height: expanded ? (longAppName ? 345 : 325) : (longAppName ? 120 : 100),
```

Comment:
Assuming we need to specify a height (which I'm guessing we do for some reason... I wish it would just grow/shrink according to the content)...

Can we always use the taller one, allowing for 2 lines of text? If that leaves odd spaces when there is only 1 line, we could use a flexbox to control the spacing so it looks nice.

Reply:
It does look a little over-large if we always use the taller one:
<img width="353" alt="Screen Shot 2019-05-29 at 9 54 50 AM" src="https://user-images.githubusercontent.com/2395211/58562745-dad7ed00-81f7-11e9-8723-2a8ea1bcca85.png">
<img width="469" alt="Screen Shot 2019-05-29 at 9 55 22 AM" src="https://user-images.githubusercontent.com/2395211/58562764-e3c8be80-81f7-11e9-9d1a-a376f00cf1d8.png">
What do you think about that?

Reply:
How does it look if you wrap it in a `div` with `display: flex, flex-direction: column` and `justify-content` of `space-around` or `space-evenly`? If that's better but still not great, there are some other flex spacing tricks we can try.